### PR TITLE
Update project to Swift 4.0

### PR DESCRIPTION
**Pull request by glennrfisher**

### CODE REVIEW

File: ChatApp/ChatApp/Credentials.swift.example [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright IBM Corporation 2016
+ * Copyright IBM Corporation 2017
```
Comment (mkistler) [inline ChatApp/ChatApp/Credentials.swift.example]: I think we probably want the copyright date to be "2016,2017".

File: README.md [hunk 2]
```diff
@@ -1,28 +1,21 @@
...
 ### Getting Started
 
 1. Clone the repository: `git clone https://github.com/watson-developer-cloud/simple-chat-objective-c.git`
-2. Build the dependencies: `carthage update --platform iOS`
-3. Open `ChatApp.xcworkspace`
-4. Copy `CredentialsExample.swift` to `Credentials.swift`.
-5. Update your service credentials in `Credentials.swift`
-6. Build and run the app!
+2. Install [Carthage](https://github.com/Carthage/Carthage) using [Homebrew](http://brew.sh/): `brew install carthage`
+3. Build the dependencies: `carthage update --platform iOS`
```
Comment (mkistler) [inline README.md:17]: Do we need `3.5 Build more dependencies: pod install`?

**Review by mkistler — APPROVED** [review]
Looks good.  Left two small suggestions that I will leave to your discretion.

File: ChatApp/ChatApp/Credentials.swift.example (hunk 1 above)
Comment (glennrfisher) [reply ChatApp/ChatApp/Credentials.swift.example]: Just pushed another commit to update the copyright date. :+1:

File: README.md (hunk 2 above)
Comment (glennrfisher) [reply README.md:17]: Looks like the `Pods` directory is checked-in here too, so it shouldn't be necessary to run a `pod install`.

**Pull request merged**